apps/cowswap-frontend/src/modules/accountProxy/containers/AccountProxyRecoverPage/styled.tsx

```diff
@@ -15,13 +15,13 @@ export const TokenWrapper = styled.div`
   display: flex;
   flex-direction: column;
   gap: 10px;
-  font-size: 14px;
+  font-size: 14px !important;
```

Comment:
I'm no css expert, but I think that's usually not a good thing?

Reply:
Yes, there are some global css that conflicts with new components.
Anyway, @fairlighteth will work on the styles, I hope he will make it perfect.